src/app/ubs-admin/components/add-violations/add-violations.component.scss

```diff
@@ -0,0 +1,179 @@
+.container {
+  font-family: var(--primary-font);
+  overflow: auto;
+}
+
+h3 {
+  font-size: 24px;
+  font-weight: 700;
+}
+
+h6 {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.close-icon {
+  color: var(--quaternary-grey);
+}
+
+.custom-radio {
+  font-size: 14px;
+}
+
+.radio {
+  color: var(--ubs-primary-green);
+  border: 1px solid var(--quaternary-grey);
+}
+
+.description {
+  width: 100%;
+  height: 141px;
+  border: 1px solid var(--ubs-quintynary-light-grey);
+  border-radius: 4px;
+  color: var(--ubs-quintynary-light-grey);
+  padding: 8px;
+}
+
+.requirement {
+  font-weight: 400;
+  font-size: 14px;
+  color: var(--quaternary-grey);
+}
+
+label {
+  padding: 4px 8px;
+  border-radius: 8px;
+}
+
+.label-y {
+  background-color: var(--attention-light-yellow);
+  margin-left: 10px;
+}
+
+.label-p {
+  background-color: var(--error-light-red);
+  margin-left: 10px;
+}
+
+.label-g {
+  color: var(--primary-green);
+}
+
+.cancel-button {
+  color: var(--primary-green);
+  border-radius: 4px;
+  border: 1px solid var(--primary-light-green);
+  background-color: #fff;
+  padding: 8px 24px;
+}
+
+.add-button {
+  color: #fff;
+  background-color: var(--primary-light-green);
+  border-radius: 4px;
+  border: none;
+  margin-left: 20px;
+  padding: 8px 24px;
+}
+
+.info {
+  color: var(--quaternary-grey);
+  font-weight: normal;
+  font-size: 12px;
+}
+
+.date {
+  color: var(--ubs-quaternary-dark-grey);
+  font-size: 14px;
+  margin-right: 30px;
+}
+
+.user {
+  color: var(--ubs-quaternary-dark-grey);
+  font-size: 14px;
+}
+
+input[type='file'] {
```

Comment:
you can use bootstrap d-none class instead

Reply:
done